### DISCUSSION TITLE
Incorporating Annealing Twin Structures

### DISCUSSION THREAD

**Discussion by janzenchoi** [neperfepx/neper#499] [Q&A] [answered]
Hello.

I was wondering whether it was possible to incorporate annealing twin structures into a volume in Neper. The closest thing I have found is to generate laminar structures to imitate the twins, though the laminar structures seem to be of equal widths, which is insufficient.

Thanks in advance.

**Comment by rquey** [comment, marked as answer]
A lamellar structure seems to be appropriate. You can specify different lamella widths as follows:
```
neper -T -n from_morpho -morpho "lamellar(w=0.1:0.3)" -o d
neper -V d.tess -datacellcol lam -print img
```
![image](https://user-images.githubusercontent.com/2659376/174573185-bd91161d-4a8e-430c-9116-97f9dcca9753.png)

**Reply by janzenchoi** [reply]
Hello rquey,

Thank you very much for the response. Do you know whether there is a way to define the width of the lamellars via a distribution, or to define the orientation of the lamellars with respects to the parent grain?

**Reply by rquey** [reply]
See https://github.com/neperfepx/neper/discussions/389#discussioncomment-2286191.
